man/systemd.resource-control.xml

```diff
@@ -691,6 +691,19 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>AllowBindPorts=<replaceable>IP_PORT</replaceable></varname></term>
+
+        <listitem>
+          <para>Allow binding socket address only to certain ip ports specified by <replaceable>IP_PORT</replaceable>.
```

Comment:
nit: I feel like there's a missing article between `binding` and `socket` (e.g. should it instead be `Allow binding a socket` or `Allow binding the socket`)